export/src/main/scala/uk/gov/nationalarchives/export/MetadataUtils.scala

```diff
@@ -0,0 +1,160 @@
+package uk.gov.nationalarchives.`export`
+
+import cats.effect.IO
+import doobie.implicits._
+import doobie.util.transactor.Transactor.Aux
+import doobie.{Get, Transactor}
+import software.amazon.awssdk.auth.credentials.DefaultCredentialsProvider
+import software.amazon.awssdk.regions.Region
+import software.amazon.awssdk.services.rds.RdsUtilities
+import software.amazon.awssdk.services.rds.model.GenerateAuthenticationTokenRequest
+import Main.Config
+import MetadataUtils._
+
+import java.util.UUID
+
+class MetadataUtils(config: Config) {
+  private def dbPassword: String = {
+    if (config.db.useIamAuth) {
+      val rdsClient = RdsUtilities.builder().region(Region.EU_WEST_2).build()
+      val port = config.db.port
+      val request = GenerateAuthenticationTokenRequest
+        .builder()
+        .credentialsProvider(DefaultCredentialsProvider.builder().build())
+        .hostname(config.db.host)
+        .port(port)
+        .username(config.db.user)
+        .region(Region.EU_WEST_2)
+        .build()
+      rdsClient.generateAuthenticationToken(request)
+    } else {
+      config.db.password
+    }
+  }
+
+  val transactor: Aux[IO, Unit] = {
+    val suffix = if (config.db.useIamAuth) {
+      s"?ssl=true&sslrootcert=/home/consignment-export/eu-west-2-bundle.pem&sslmode=verify-full"
```

Comment:
Been told not to use `sslmode=verify-full`. Think should be using `sslmode=require`?